pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3742,6 +3743,22 @@ func Test_server_MultiInsert(t *testing.T) {
 		return req
 	}
 
+	genNInsertReq := func(num int, vec []float32) *payload.Insert_MultiRequest {
```

Comment:
It should be
```suggestion
	genMInsertReq := func(num int, vec []float32) *payload.Insert_MultiRequest {
```
or
```suggestion
	genMultiInsertReq := func(num int, vec []float32) *payload.Insert_MultiRequest {
```

Reply:
I suppose it would be better
```go
	genMultiInsertReq := func(num int, type string) *payload.Insert_MultiRequest {
             // create vector
             // create request
        }
```

Reply:
duplicated implementation?
https://github.com/vdaas/vald/blob/fb5df3c3e355b9158ac2485b1701d7ef2ec5cabf/pkg/agent/core/ngt/handler/grpc/handler_test.go#L3710-L3726

Reply:
do you mean it is duplicated with genIntReq? 
we can consider separate the create multiInsert request(line 3713-3725) to another function 🤔 
What do you think?

Reply:
I think this will be fixed by this refactoring 
https://github.com/vdaas/vald/pull/1645#discussion_r870980778

Reply:
it is not correct.
```
I suppose it would be better

	genMultiInsertReq := func(num int, type string) *payload.Insert_MultiRequest {
             // create vector
             // create request
        }
```
`genMultiInsertReq` generate multiInsert request with the same vector, so it should be OK with the current implement.